### PR TITLE
Implement design for new aquisitions

### PR DESCRIPTION
**Pull request by snaegle**
Related to: AAC-511

### CODE REVIEW

File: Resources/Private/Scss/App.scss [hunk 1]
```diff
@@ -29,6 +30,7 @@
 @import 'ezbrequest';
 @import 'find';
 @import 'solr';
+@import 'newAquisitions';
```
Comment (nilswindisch) [inline Resources/Private/Scss/App.scss:33]: We really should agree on one scheme… camelCase or hyphen?

File: Resources/Private/Scss/_newAquisitions.scss [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+    }
+
+    // the anchor is clickable
+    a.pz2-prev,
+    a.pz2-next {
+        color: $shadeDarkest;
+        padding: 0;
+
+        svg {
+            width: 1.5em;
```
Comment (nilswindisch) [inline Resources/Private/Scss/_newAquisitions.scss:205]: don't we usually avoid using ems?

File: Resources/Private/Scss/elements/form-checkbox.scss [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    input:not(:checked) + label::after {
+        content: '';
+        width: 10px;
+        height: 10px;
+        background: $shadeDarkest;
+        position: absolute;
+        top: 4px;
+        left: 4px;
+        border-radius: 0;
+        transition: all 0.2s ease;
```
Comment (nilswindisch) [inline Resources/Private/Scss/elements/form-checkbox.scss:41]: @MustafaSalih introduced a variable for transition values here https://github.com/subugoe/typo3-tmpl_fidaac/commit/723f56fab5632a18f0c8537c06c6909543ab670a

**Comment by nilswindisch** [comment]
The way items are selected acts really awkward… or is it the intended behaviour?

(video via email)

File: Resources/Private/Scss/App.scss (hunk 1 above)
Comment (snaegle) [reply Resources/Private/Scss/App.scss:33]: We should consolidate that in another issue

**Comment by snaegle** [comment]
Pazpar2 stores the value of the checkboxes in a cookie. The results are incoherent, you are right. I would just suggest to tackle that in a different issue as it takes more time to solve.

**Comment by snaegle** [comment]
Nice, to use a variable. Should be used everywhere, not just here.

**Pull request merged**